### PR TITLE
Addition of a new class to LL19, SKARL13, BCHYDRO gmms.

### PR DESCRIPTION
**Pull request by mascandola**

### CODE REVIEW

**Review by mmpagani — CHANGES_REQUESTED** [review]

File: openquake/hazardlib/gsim/abrahamson_2015.py [hunk 1]
```diff
@@ -459,3 +459,62 @@ class AbrahamsonEtAl2015SSlabLow(AbrahamsonEtAl2015SSlab):
...
+
+
+class AbrahamsonEtAl2015SInter_scaled(AbrahamsonEtAl2015SInter):
+    """
+    Implements the Subduction GMPE developed by Norman Abrahamson, Nicholas
+    Gregor and Kofi Addo, otherwise known as the "BC Hydro" Model, published
+    as "BC Hydro Ground Motion Prediction Equations For Subduction Earthquakes
+    (2015, Earthquake Spectra, in press), for subduction interface events.
+
+    Application of a scaling factor that converts the prediction of
```
[the remark this answers is not in the document]
Comment (mascandola) [reply openquake/hazardlib/gsim/abrahamson_2015.py:471]: I do not have a reference explaining this version.
This model is for the maximum ground motion value of the horizontal components.
It is computed by correcting the offset value for a scaling factor.

File: openquake/hazardlib/gsim/abrahamson_2015.py [hunk 2]
```diff
@@ -459,3 +459,62 @@ class AbrahamsonEtAl2015SSlabLow(AbrahamsonEtAl2015SSlab):
...
+	0.75	410.5	-2.401	3.347462884	-0.91	-0.0058		0.3687	-0.54	1.42	0.012	1.993	-0.0296	-0.12	0.48	-0.34	0.6	0.43	0.74	0.6
+	1		400		-1.955	2.922350962	-0.85	-0.0062		0.1746	-0.34	1.1		0.0114	1.47	-0.0363	-0.07	0.33	-0.14	0.6	0.43	0.74	0.6
+	2		400		-0.299	1.539520753	-0.71	-0.0064		-0.2821	0.12	0.7		0.0085	-0.401	-0.061	0		0.3		0		0.6	0.43	0.74	0.6
+	3		400		0		0.776117425	-0.64	-0.0064		-0.4466	0.3		0.7		0.0054	-0.673	-0.0798	0		0.3		0		0.6	0.43	0.74	0.6
+	4		400		0		0.202413823	-0.58	-0.0064		-0.4344	0.3		0.7		0.0027	-0.627	-0.0935	0		0.3		0		0.6	0.43	0.74	0.6
+
+    """)
+
+
+class AbrahamsonEtAl2015SSlab_scaled(AbrahamsonEtAl2015SInter_scaled):
```
[the remark this answers is not in the document]
Comment (mascandola) [reply openquake/hazardlib/gsim/abrahamson_2015.py:496]: To modify the "AbrahamsonEtAl2015SSlab" slab model I had to modify also the "AbrahamsonEtAl2015SInter" Interface model, which is recalled by the former.

File: abrahamson_2015_test.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    MEAN_FILE = "BCHYDRO/BCHYDRO_SINTER_CENTRAL_scaled_MEAN.csv"
+
+    def test_all(self):
+        self.check(self.MEAN_FILE,
+                   max_discrep_percentage=0.1)
+
+
+class AbrahamsonEtAl2015SSlabScaledTestCase(BaseGSIMTestCase):
+    """
+    Tests the Abrahamson et al. (2015) BC Hydro model for subduction
```
[the remark this answers is not in the document]
Comment (mascandola) [reply abrahamson_2015_test.py:141]: We have prepared the verification tables from the corresponding Matlab code.

File: openquake/hazardlib/gsim/lanzano_luzi_2019.py [hunk 4]
```diff
@@ -192,3 +192,49 @@ def compute(self, ctx, imts, mean, sig, tau, phi):
...
+    flank movements (focal depths <5km) and deeper events occurring due to
+    regional tectonics (focal depths >5km), considering two different
+    attenuations with distances.
+
+    Test tables are generated from a spreadsheet provided by the authors, and
+    modified according to OQ format (e.g. conversion from cm/s2 to m/s2).
+    
+    Application of a scaling factor that converts the prediction of
+    LanzanoLuzi2019shallow to the corresponding
+    prediction for the Maximum value.
```
[the remark this answers is not in the document]
Comment (mascandola) [reply openquake/hazardlib/gsim/lanzano_luzi_2019.py:216]: This model is for the maximum ground motion value of the two horizontal components.

File: openquake/hazardlib/gsim/lanzano_luzi_2019.py [hunk 5]
```diff
@@ -192,3 +192,49 @@ def compute(self, ctx, imts, mean, sig, tau, phi):
...
+	1		-2.463138493	1.1553	-1.723	-1.5615	-0.0018	0.1201	0.448	0.1496	0.2279	0.1904	0.297	0.3325
+	2		-3.172265876	1.1995	-1.6524	-1.6597	-0.0009	0.144	0.3917	0.1929	0.2187	0.1824	0.2848	0.344
+	3		-3.65485242		1.2118	-1.5741	-1.6063	-0.0012	0.1261	0.3836	0.2356	0.2139	0.1825	0.2812	0.3668
+	4		-3.445025941	1.0943	-1.4949	-1.6025	-0.0012	0.1064	0.3447	0.2442	0.2093	0.1832	0.2782	0.3701
+	pgv		-2.493041253	0.9809	-1.8482	-1.5676	-0.0042	0.0995	0.3747	0.1433	0.2126	0.2099	0.2988	0.3313
+	""")
+
+
+class LanzanoLuzi2019deep_scaled(LanzanoLuzi2019shallow_scaled):
+    kind = 'deep'
```
[the remark this answers is not in the document]
Comment (mascandola) [reply openquake/hazardlib/gsim/lanzano_luzi_2019.py:240]: ok, we have modified the file following the original version on OQ (not scaled version).

**Comment by mmpagani** [comment]
LGTM, thanks Claudia for your comments

**Pull request merged**